deploy/packages/rpm/Makefile

```diff
@@ -22,6 +22,8 @@ CURDIR := $(shell pwd)
 
 POST_ADDITION := $(if $(SYSTEMD),%systemd_post %{_name}.service,echo post)
 PREUN_ADDITION := $(if $(SYSTEMD),%systemd_preun %{_name}.service,echo preun)
```

Comment:
we can remove the `SYSTEMD` at line 19 and remove the `if` condition here at line 23 and 24.

Reply:
Great, Done!